### PR TITLE
Update menu.json

### PR DESCRIPTION
**Pull request by bbrala**
jQuery contextmenu link is quite outdated :) We are the new maintainer.

### CODE REVIEW

File: features-json/menu.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "title":"Demo"
     },
     {
-      "url":"http://addyosmani.github.com/jQuery-contextMenu/",
+      "url":"https://github.com/swisnl/jQuery-contextMenu,
```
Comment (cvrebert) [inline features-json/menu.json]: Missing closing quotation mark here

**Comment by bbrala** [comment]
I really shouldn't do this late at night, sorry to waste your time.

**Comment by Fyrd** [comment]
Thanks!

**Comment by bbrala** [comment]
No problem, thanks for the quick responses :)

**Pull request merged**